build_support/windows_cross/cross_tool.py

```diff
@@ -189,9 +207,19 @@ def get_config(arch, required_tool, ignore_cache):
     if not tool_path:
         print(f"tool {tool} not found, try installing it")
         exit(1)
-    lib = ';'.join(config['lib'])
-    include = ';'.join(config['include'])
-    environ = dict(os.environ.copy(), LIB=lib, INCLUDE=include)
+
+    separator = ':' if tool == "msvc-midlrt" else ';'
+    lib = separator.join(config['lib'])
+    include = separator.join(config['include'])
+    path = separator.join(config['path'])
+    environ = dict(os.environ.copy(), LIB=lib, INCLUDE=include, PATH=path)
+    if tool == "msvc-midlrt":
+        wslenv = environ['WSLENV']
+        if wslenv is None:
+            wslenv = ""
+        wslenv = wslenv + ":PATH/wp:INCLUDE/wp:LIB/wp"
```

Comment:
I don't think we need the extra path stuff as long as we pass /no_cpp to midlrt (which I think we can since we don't import any idl files from the SDK).